docs/store/datastore.md

```diff
@@ -115,4 +106,64 @@ be used if this file is located in a location on shared storage, accessible to p
 * `GCP_CREDENTIALS` &mdash; when the credentials file cannot be mounted to the pod, this secret or environment variable
 may contain the contents of this file. If configured in the function pod, MLRun dumps its contents to a temporary file 
 and points `GOOGLE_APPLICATION_CREDENTIALS` at it. An exception is `BigQuerySource`, which passes `GCP_CREDENTIALS`'s
-contents directly to the query engine.
\ No newline at end of file
+contents directly to the query engine.
+
+### S3
+* `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` &mdash; [access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html)
+  parameters
+* `S3_ENDPOINT_URL` &mdash; the S3 endpoint to use. If not specified, it defaults to AWS. For example, to access 
+  a storage bucket in Wasabi storage, use `S3_ENDPOINT_URL = "https://s3.wasabisys.com"`
+* `MLRUN_AWS_ROLE_ARN` &mdash; [IAM role to assume](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-api.html). 
+  Connect to AWS using the secret key and access key, and assume the role whose ARN is provided. The 
+  ARN must be of the format `arn:aws:iam::<account-of-role-to-assume>:role/<name-of-role>`
+* `AWS_PROFILE` &mdash; name of credentials profile from a local AWS credentials file. 
+  When using a profile, the authentication secrets (if defined) are ignored, and credentials are retrieved from the 
+  file. This option should be used for local development where AWS credentials already exist (created by `aws` CLI, for
+  example)
+  
+## Using data store profiles
+
+You can use a data store profile to manage datastore credentials. A data store profile 
+holds all the information required to address an external data source, including credentials. 
+You can create 
+multiple profiles for one datasource, for example, 
+two different Redis data stores with different credentials. Targets, sources, and artifacts, 
+can all use the data store profile by using the `ds://<profile-name>` convention.
+After you create a profile object, you make it available on remote pods by calling 
+`project.register_datastore_profile`.
+
+Create a data store profile in the context of a project. Example of creating a Redis datastore profile:
+1. Create the profile, for example:<br>
+   `profile = DatastoreProfileRedis(name="test_profile", endpoint_url="redis://11.22.33.44:6379", username="user", password="password")`
+   The username and password parameters are optional. 
```

Comment:
new sentence

Reply:
@assaf758 Looks like you mean there's no period at the end of the profile=...
But the sentence begins with a capital letter and on a new line, so it's obvious. If you mean something else, please explain.